### DISCUSSION TITLE
generic url for latest release zip file

### DISCUSSION THREAD

**Discussion by wdlarg01** [ComplianceAsCode/content#10942] [Q&A] [answered]
For starters, thanks to all the contributors here for phenomonal work. I'm a n00b dinosaur and these efforts just astound me.

Making sure I'm not missing some info.
I know I can use the "/latest" url to get to the latest release for manual downloads. I'd like to auto mate things (via ansible) and was wondering if there was some form of "latest.zip" url I can give to wget (or ansible get_url).

Without this, my plans are to parse "curl -D" and work out from there, unless there are other ideas.

I prefer the ability to use the native builds here for consistency across our multi-platform farm.

Thanks for the consideration.

**Comment by ggbecker** [comment, marked as answer]
I believe there is no easy way to do that. We workaround here for example:

https://github.com/OpenSCAP/scap-workbench/pull/302/files#diff-d0777657fa3fd81d23aaf7273e58aee453b04e67882517900c56daeef9b3e4c1R78-R80

```
        REL_TAG=`wget -q -O- https://api.github.com/repos/ComplianceAsCode/content/releases/latest | jq -r '.tag_name'`
        REL_TAG_NUM=`echo ${REL_TAG} | cut -d"v" -f2`
        DWN_LINK="https://github.com/${REPOSITORY}/releases/download/${REL_TAG}/scap-security-guide-${REL_TAG_NUM}.zip"
```
echoing DWN_LINK for me gives at this moment the following:

```
$ echo $DWN_LINK
https://github.com//releases/download/v0.1.69/scap-security-guide-0.1.69.zip
```

**Reply by wdlarg01** [reply]
Thanks. kinda what I figured. You've done basically what I was thinking. My scripting skills are not as developed lol.

**Reply by wdlarg01** [reply]
For the benefit of others thay may ask, your response does work in an ansible playbook.
Far simpler than what I was coming up with.
Thanks again.

    - name: get redirect version
      shell: "wget -q -O- https://api.github.com/repos/ComplianceAsCode/content/releases/latest | jq -r '.tag_name'"
      register: verstring

    - name: show value
      debug:
        msg: "{{verstring.stdout}}"
